bundle/manifests/lifecycle-agent.clusterserviceversion.yaml

```diff
@@ -21,8 +21,16 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/lifecycle-agent-operator
+    description: The Lifecycle Agent Operator for OpenShift provides local (on cluster)
+      lifecycle management services for Single Node Openshift (SNO) clusters.
+    features.operators.openshift.io/disconnected: "true"
```

Comment:
I am not 100% sure, but I recall we are supposed to explicitly specify all other features as false.